lib/utils.py

```diff
@@ -12,6 +12,22 @@
 
 CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
 
+def is_outside_symlink(base_dir, symlink_path):
+    try:
+        abs_target = os.path.realpath(symlink_path)
+        return not abs_target.startswith(os.path.realpath(base_dir)), abs_target
+    except OSError:
+        return False, None  # Not a symlink
```

Comment:
I also found the `os.path.commonpath` which might be a little better than `startswith` 

Also the `path` has a `resolve()` which resolves all symlinks.